effect/shared/src/main/scala/scalaz/effect/AsyncReturn.scala

```diff
@@ -17,8 +17,9 @@ object AsyncReturn {
   object Later {
     // Trick thanks to Sam Halliday. We eliminate allocation overhead but also
```

Comment:
it's really @xuwei-k's trick... mine was cleaner but broke exhaustivity checking. And @hrhino came up with the fix for the dead code thing.

BTW, can we please make @hrhino a member of the org so he can approve PRs?

Reply:
I think I am a member? so I'm confused.

Reply:
@hrhino oh, when I see your reviews they are often grayed out... whereas others come through with a green tick.

Reply:
Yep he's a member already.